src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -24,6 +26,19 @@ public SnapshotBuilderSettings updateSnapshotBuilderSettings(
     return snapshotBuilderSettingsDao.upsertSnapshotBuilderSettingsByDataset(id, settings);
   }
 
+  public JobModel requestSnapshot(
```

Comment:
I don't think this will need to return a job. Jobs are mostly for asynchronous operations, and I think requesting a snapshot is going to be synchronous. I would just have this return the request made for now, personally.

When we actually get around to an admin accepting the request, that will be asynchronous, and that one will need a job.

Reply:
Okay, that makes sense. I was returning the request originally. Then, I talked to Phil and I decided to change it to return a fake id to mock the behavior of when the request actually gets posted to the database. I decided to use a job because I saw another create method do something similar. Instead, I could return an id without using a job

Reply:
Sorry for the misleading advice! Yes I was thinking about the point where the snapshot creation begins, which doesn't happen until the request is approved.